### PR TITLE
Cvsl 2121 afer initial outline of view assessment page

### PR DESCRIPTION
**Pull request by nishanthmahasamudram**

### CODE REVIEW

File: server/data/__testutils/testObjects.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+import type { AssessmentSummary, OffenderSummary } from '../../@types/assessForEarlyReleaseApiClientTypes'
+import AssessmentStatus from '../../enumeration/assessmentStatus'
 
 const createOffenderSummary = ({
   prisonNumber = 'A1234AB',
   bookingId = 54321,
   forename = 'Jim',
   surname = 'Smith',
-  hdced = '2022-08-01',
+  hdced = '01 Aug 2022',
```
Comment (andrewrlee) [inline server/data/__testutils/testObjects.ts:9]: **question:** isn't the api returning dates in yyyy-mm-dd format?

Comment (nishanthmahasamudram) [reply server/data/__testutils/testObjects.ts:9]: Yes, it is returning in yyyy-mm-dd.
I am using this as response and updating request with the above format in the test files.

File: server/services/caseAdminCaseloadService.test.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import { CaseAdminCaseloadService } from '.'
 import { createAssessForEarlyReleaseApiClient } from '../data/__testutils/mocks'
-import { createOffenderSummary } from '../data/__testutils/testObjects'
+import { createAssessmentSummary, createOffenderSummary } from '../data/__testutils/testObjects'
```
Comment (andrewrlee) [inline server/services/caseAdminCaseloadService.test.ts:3]: **question:** This isn't quite right is it? these return dates in the wrong format? 

Should be iso format from the backend?

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**